### PR TITLE
Vue/Spanish

### PR DESCRIPTION
**Pull request by IvanSotelo**
This pull request update the spanish translation for the learnstorybook for Vue.

### CODE REVIEW

**Comment by IvanSotelo** [comment]
PR #435

**Comment by jonniebigodes** [comment]
@IvanSotelo Glad to hear from you again. Hope you're doing good in these times of uncertainty. Before we dive into this i'm currently wrapping up some changes needed to the tutorial that follow up #437. Once it's up i'll let you know and so that you can adjust accordingly and we get it proofread and merged. Do you mind waiting a little bit so that i can wrap up my changes ?

**Comment by jonniebigodes** [comment]
@IvanSotelo when you get the chance could you update your fork/branch. As you may notice there's some files that have conflicts. This is based on #453 that i recently merged addressing a bit of an issue we had when we consolidated the entire documentation. I'm going to take a look at the content during the weekend and if everything is ok we should be able to get it merged sometime next week. 

Let me know once you've taken care of it and we'll go from there. 

Stay safe

**Comment by jonniebigodes** [comment]
@IvanSotelo if you could adjust your pull request and factor in the changes introduced by #456 i would really appreciate it, as there were a couple of issues that were factored in. Let me know once the changes are up and i'll take another look. Sounds good?

Stay safe

**Review by amartinez62 — COMMENTED** [review]
exportacion should be exportación

"Por lo tanto, para evitar este problema, podemos usar Jest para renderizar la historia en el DOM y ejecutar algún código de consulta del DOM para verificar las características salientes del resultado. Lo bueno del formato de la historia es que simplemente podemos importar la historia en nuestras pruebas y reproducirla allí." renderizar should be hacer.

File: content/intro-to-storybook/vue/es/composite-component.md [hunk 1]
```diff
@@ -23,125 +23,96 @@ Un componente compuesto no es muy diferente de los componentes básicos que cont
...
+  // Shaping the stories through args composition.
+  // Inherited data coming from the Loading story.
+  ...Loading.args,
+  loading: false,
+};
 ```
 
 <div class="aside">
     Los <a href="https://storybook.js.org/docs/vue/writing-stories/decorators"><b>Decoradores</b></a> son una forma de proporcionar envoltorios arbitrarios a las historias. En este caso estamos usando un decorador en la exportacion predeterminada para añadir estilo. También se pueden usar para agregar otro contexto a los componentes, como veremos más adelante.
 </div>
```
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/es/composite-component.md:113]: @IvanSotelo add the 💡 before "Los decoradores" turning this into:
```html
<div class="aside">
💡 Los <a href="https://storybook.js.org/docs/vue/writing-stories/decorators"><b>Decoradores</b></a> .....
</div>
```

File: content/intro-to-storybook/vue/es/deploy.md [hunk 2]
```diff
@@ -1,100 +1,138 @@
...
-<div class="aside">
-<strong>¿Hiciste los test con Chromatic antes?</strong>
-<br/>
-🎉 Sus historias ya están desplegadas! Chromatic indexa de forma segura sus historias en línea y las rastrea a través de ramas y commits. Salta este capítulo y ve a la <a href="/intro-to-storybook/vue/es/conclusion">conclusión</a>.
-</div>
+A lo largo de este tutorial, creamos componentes en nuestra máquina de desarrollo. En algún momento, tendremos que compartir nuestro trabajo para recibir comentarios del equipo. Despleguemos Storybook en línea para ayudar a los compañeros de equipo a revisar la implementación de la interfaz de usuario.
 
 ## Exportando como una app estática
 
 Para desplegar Storybook primero necesitamos exportarlo como una aplicación web estática. Esta funcionalidad ya está incorporada en Storybook, solo tenemos que cambiarla como lo hicimos antes cuando el proyecto se inicializó en la [sección de Empezando](/intro-to-storybook/vue/es/get-started).
```
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/es/deploy.md:13]: @IvanSotelo if you could adjust this to :

```markdown
Para desplegar Storybook primero necesitamos exportarlo como una aplicación web estática. Esta funcionalidad ya está incorporada en Storybook.
```

It generated a redundancy as we're mentioning the same chapter below

File: content/intro-to-storybook/vue/es/using-addons.md [hunk 3]
```diff
@@ -5,166 +5,86 @@ description: 'Aprende a integrar y usar complementos usando un ejemplo popular'
 commit: '45b6600'
 ---
 
-Storybook cuenta con un sistema robusto de [complementos](https://storybook.js.org/docs/vue/configure/storybook-addons) con el que puede mejorar la experiencia del desarrollador para
-todos en tu equipo. Si ha seguido este tutorial linealmente, hasta ahora hemos hecho referencia a varios complementos, y ya habrá implementado uno en el [Testing](/intro-to-storybook/vue/es/test/).
+Storybook cuenta con un sistema robusto de [complementos](https://storybook.js.org/docs/vue/configure/storybook-addons) con el que puede mejorar la experiencia del desarrollador para todos en tu equipo.Véalos todos [aquí](https://storybook.js.org/addons).
```
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/es/using-addons.md:8]: @IvanSotelo could you add a space here turning this into:

"
... con el que puede mejorar la experiencia del desarrollador para todos en tu equipo. Véalos ...."

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@IvanSotelo just checked the pull request and we're almost in the clear and get this merged. Left a couple of minor items that require attention and if you could factor in the change that @amartinez62 left we'd appreciate it.


On that note,  @amartinez62 thank you very much for taking the time to review the pull request and help polish the wording. Really appreciate it.


Let me know once the changes are in and we'll get this "out the door" (pardon the bad pun)

**Review by amartinez62 — COMMENTED** [review]
LGTM! 🎉

**Comment by jonniebigodes** [comment]
@IvanSotelo looks like @amartinez62 gave the thumbs up! I'm going to take a look at it tomorrow and if anything else stands out. This will be merged by Monday at the latest.

**Comment by jonniebigodes** [comment]
All good from your end @IvanSotelo! Merging this! I'll need to follow up on it. But once again and it goes without saying, thank you for the time and effort you've put into this and for addressing the feedback so promptly! 

Stay safe

**Pull request merged**